app/src/main/java/org/p2p/wallet/common/AppSettings.kt

```diff
@@ -0,0 +1,25 @@
+package org.p2p.wallet.common
+
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import org.p2p.wallet.BuildConfig
+
+private const val KEY_POLLING_ENABLED = "KEY_POLLING_ENABLED"
+private const val KEY_IS_PROD = "KEY_IS_PROD"
+
+class AppSettings(private val sharedPreferences: SharedPreferences) {
+
+    val isPoolingEnabled: Boolean
+        get() = sharedPreferences.getBoolean(KEY_POLLING_ENABLED, true)
+
+    fun setPollingEnabled(isEnabled: Boolean) {
+        sharedPreferences.edit { putBoolean(KEY_POLLING_ENABLED, isEnabled) }
+    }
+
+    val isProd: Boolean
```

Comment:
isProduction / isReleaseBuild

Reply:
isDevNetEnabled 😉

Reply:
nice!

Reply:
Mb isDevnetEnabled? 
Otherwise it looks sOmetHinGLiKe tHaT )

Reply:
as you wish

Reply:
as you wish

Reply:
yes i will
i will ping you
making changes right now

Reply:
@1sakov you can check gradle please